contrib/test_mergeBenchmarkSets/test_mergeBenchmarkSets.py

```diff
@@ -0,0 +1,538 @@
+# This file is part of BenchExec, a framework for reliable benchmarking:
+# https://github.com/sosy-lab/benchexec
+#
+# SPDX-FileCopyrightText: 2007-2020 Dirk Beyer <https://www.sosy-lab.org>
+#
+# SPDX-License-Identifier: Apache-2.0
+
+import copy
+import sys
+import unittest
+import xml.etree.ElementTree as ET  # noqa: What's wrong with ET?
+
+from contrib import mergeBenchmarkSets
+from benchexec import result
+
+sys.dont_write_bytecode = True  # prevent creation of .pyc files
+
+mock_results = """<?xml version="1.0"?>
```

Comment:
Wouldn't it make the tests easier to read if these were separate files?

Reply:
I suppose it would -> changed.